### PR TITLE
Register addons at the beginning of Public Cloud QAM scenario

### PR DESCRIPTION
**Pull request by pdostal**
Register all the addons just after the system in `transfer_repos.pm` and do not remove those in following tests.

- Related ticket: [poo#54842](https://progress.opensuse.org/issues/54842)
- Needles: No need
- Verification run: [12sp4@ec2](http://pdostal-server.suse.cz/tests/5864) [12sp4@gce](http://pdostal-server.suse.cz/tests/5863) [15sp1@ec2](http://pdostal-server.suse.cz/tests/5860) [15sp1@gce](http://pdostal-server.suse.cz/tests/5862)

### CODE REVIEW

File: tests/publiccloud/transfer_repos.pm [hunk 1]
```diff
@@ -20,10 +20,18 @@ use utils;
 
 sub run {
     my ($self, $args) = @_;
+
+    set_var('SCC_ADDONS', '') unless get_var('SCC_ADDONS');
+    my @addons = split(/,/, get_var('SCC_ADDONS'));
```
Comment (dzedro) [inline tests/publiccloud/transfer_repos.pm]: You can just use default value `get_var('SCC_ADDONS', '')`

**Comment by pdostal** [comment]
Verification runs are fresh and green and branch has been rebased.

**Comment by dzedro** [comment]
Why are you adding only sdk and not all modules same as on other exratests on this version ?

**Comment by pdostal** [comment]
> Why are you adding only sdk and not all modules same as on other exratests on this version ?

This was agreed for the beginning. When this will be stable, the scheduling will probably be modified.

**Pull request merged**